src/main/java/org/threadly/litesockets/buffers/AbstractMergedByteBuffers.java

```diff
@@ -58,20 +58,15 @@ public void add(final byte[] ...bas) {
   @Override
   public void add(final ByteBuffer ...buffers) {
     for(ByteBuffer buffer: buffers) {
-      if(buffer.hasRemaining()) {
```

Comment:
doAppend is the only place we can do this check and 100% verify that empty buffers are never added.  Granted it would likely be a user submitting an empty buffer to cause it, but I just thought it would be best if we could 100% be sure there are no empty buffers.